components/alert/alert.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Decora</title>
+    <link rel="stylesheet" href="../../main.css">
+    <link rel="stylesheet" href="../../sideNav.css">
+    <link rel="stylesheet" href="./alert.css">
+    <link rel="stylesheet" href="../component.css">
```

Comment:
Instead of importing files in HTML file. Create a separate CSS file, import all the files in that single CSS file and import that single file everywhere.

Reply:
Noted. On it.:+1: